tutorials/dev-app-cli.adoc

```diff
@@ -0,0 +1,162 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="dev-app-cli"]
+= Tutorial: Deploying an application by using the CLI
+include::_attributes/common-attributes.adoc[]
+:context: dev-app-cli
+
+toc::[]
+
+This tutorial walks you through deploying services to stand up an application called `national-parks-app` on {product-title} that displays a map of national parks across the world. You will use the {oc-first} to complete this tutorial.
+
+To complete this tutorial, you will perform the following steps:
+
+. xref:../tutorials/dev-app-cli.adoc#getting-started-cli-creating-new-project_dev-app-cli[Create a new project for the application].
++
+This step allows your application to be isolated from other cluster user's workloads.
+
+. xref:../tutorials/dev-app-cli.adoc#getting-started-cli-granting-permissions_dev-app-cli[Grant view permissions].
++
+This step grants `view` permissions to interact with the OpenShift API to help discover services and other resources running within the project.
+
+. xref:../tutorials/dev-app-cli.adoc#getting-started-cli-deploying-first-image_dev-app-cli[Deploy the front-end application].
++
+This step deploys the `parksmap` front-end application, exposes it externally, and scales it up to two instances.
+
+. xref:../tutorials/dev-app-cli.adoc#getting-started-cli-deploying-python-app_dev-app-cli[Deploy the back-end application].
++
+This step deploys the `nationalparks` back-end application and exposes it externally.
+
+. xref:../tutorials/dev-app-cli.adoc#getting-started-cli-connecting-database_dev-app-cli[Deploy the database application].
++
+This step deploys the `mongodb-nationalparks` MongoDB database, loads data into the database, and sets up the necessary credentials to access the database.
+
+. xref:../tutorials/dev-app-cli.adoc#getting-started-cli-view_dev-app-cli[Verify that the map application is available].
++
+This step verifies that you can view the map of national parks in a browser.
+
+[id="before-you-begin_{context}"]
+== Before you begin
+
+Before you start this tutorial, ensure that you have the the following required prerequisites:
```

Comment:
```suggestion
Ensure that you have the the following required prerequisites:
```
I think this can be simplified since the name of the section is "Before you begin".

Reply:
I renamed the heading to "Prerequisites". That's more in line with what our mod docs templates say anyways.